packages/sdk/src/streamStateView_Members_Solicitations.ts

```diff
@@ -32,19 +32,19 @@ export class StreamStateView_Members_Solicitations {
         user.solicitations = user.solicitations.filter(
             (x) => x.deviceKey !== solicitation.deviceKey,
         )
-        user.solicitations.push({
+        const newSolicitation = {
             deviceKey: solicitation.deviceKey,
             fallbackKey: solicitation.fallbackKey,
             isNewDevice: solicitation.isNewDevice,
             sessionIds: [...solicitation.sessionIds.sort()],
```

Comment:
sort mutates the original array, [...solicitation.sessionIds].sort() ?

Reply:
i guess why this wasn't a problem, i'll fix